app/views/phylo_trees/index.html.erb

```diff
@@ -5,6 +5,7 @@
       taxon: JSON.parse('<%= raw escape_json(@taxon) %>'),
       phyloTrees: JSON.parse('<%= raw escape_json(@phylo_trees) %>'),
       allowedFeatures: JSON.parse('<%= raw escape_json(@current_user.allowed_feature_list) %>'),
-    }, 'PhyloTreeListView_page');
+    }, 'PhyloTreeListView_page',
+    JSON.parse('<%= raw escape_json(request_context)%>'));
```

Comment:
Why did you need to add this on this PR?

Reply:
wasn't necessary 👍 probably could've bundled with a different PR

Reply:
OK, fine to change if you want to bundle it...